### DISCUSSION TITLE
How to install or enable xdebug ?

### DISCUSSION THREAD

**Discussion by SajithAthukorala** [shivammathur/homebrew-php#599] [Q&A / Help] [answered]
I used this method to switch my php versions , i want to know how to install or enable xdebug with my older php version which i use ? I used php version 7.0

**Comment by shivammathur** [comment, marked as answer]
I would recommend using the [`shivammathur/extensions`](https://github.com/shivammathur/homebrew-extensions) tap for this.
```zsh
brew tap shivammathur/extensions
brew install xdebug@7.0
```

**Reply by SajithAthukorala** [reply]
Wow this worked perfectly  , thanks a lot , do you mind to guide me to enable debugging xdebug with php storm in symfony project , never mind if your not know about it !!!
You  saved many hours already thanks a lot !!!

**Reply by shivammathur** [reply]
This [article](https://medium.com/@ahmetmertsevinc/the-easiest-way-to-debug-a-symfony-application-in-phpstorm-25d609e374a7) might help you.
